src/test/rustdoc/internal.rs

```diff
@@ -1,12 +0,0 @@
-// compile-flags: -Z force-unstable-if-unmarked
-
-// @matches internal/index.html '//*[@class="docblock-short"]/span[@class="stab internal"]' \
-//      'Internal'
```

Comment:
This test should be modified to check that the unstable marker is not added for `rustc_private`.

Reply:
Done